.env

```
@@ -0,0 +1,3 @@
+LINE_CHANNEL_ID = 
+LINE_CHANNEL_SECRET = 
+LINE_CHANNEL_TOKEN = 
```

```suggestion
LINE_CHANNEL_ID=
LINE_CHANNEL_SECRET=
LINE_CHANNEL_TOKEN=
```

spaceがあるとだめな気がします。違いますかね。

---

ダメではないみたい（動きました）でしたけど、詰めて書いてるのが良いですかね？